CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
 ### 🧰 Bug fixes 🧰
 
 - `kubletetstatsreceiver`: Bring back `k8s.container.name` attribute (#10848)
+- `filterprocessor`: Fix unit tests (#10886)
```

Comment:
Please remove this line, we don't usually add entries to the changelog for test changes. I've added the skip changelog label 👍🏻

Reply:
```suggestion
```